api/buf.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Apache Software Foundation (ASF) licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+version: v1beta1
+build:
+  roots:
+    - ./proto
+lint:
+  use:
+    - DEFAULT
+  except:
+    - RPC_REQUEST_STANDARD_NAME
```

Comment:
As I asked in the last PR #23 , do we need to impose these two rules?

Reply:
It's the standard/common pattern to name the request. skywalking keeps using this pattern to build grpc service.

Reply:
> It's the standard/common pattern to name the request. skywalking keeps using this pattern to build grpc service.

So if we need to comply with the conventions, we have to change the request and response types as well as delete this `except` section.

Reply:
Would you fix this issue?

Reply:
> Would you fix this issue?

Sure I can do this later.